rules/rbe_repo.bzl

```diff
@@ -1077,6 +1102,11 @@ def rbe_autoconfig(
         use_checked_in_confs = use_checked_in_confs,
     )
 
+    if force_checked_in_confs and not config_version:
```

Comment:
Consider emitting an error if use_checked_in_confs is set to False when force_checked_in_confs is set to True

Reply:
moved to using a single attr for both